include/seqan3/alignment/pairwise/alignment_algorithm.hpp

```diff
@@ -247,7 +247,7 @@ class alignment_algorithm :
     {
         assert(static_cast<size_t>(std::ranges::distance(sequences)) <= traits_t::alignments_per_vector);
 
-        using simd_score_t = typename traits_t::score_t;
+        using simd_score_t = typename traits_t::score_type;
```

Comment:
```suggestion
        using simd_score_type = typename traits_t::score_type;
```
? But I don't care.

Reply:
Well, good question. We have not defined any particular rules about this. So far I use it very inconsistent but try to establish the following rule: use `_t` if it is a template parameter of a class or function or only used in local scope of a function. Use `_type` when defined as member types of a class/struct.

Reply:
Since I wouldn't do rules for every little thing, I suggest a rule for using `_type` for public member types of a class struct (since consistency in the public API is nice), and otherwise, leave it to the author of the code.